math/mean/mean.go

```diff
@@ -0,0 +1,27 @@
+package mean
```

Comment:
You do not need to create a separate package just for one function. This clearly belongs in the `math` package.

Reply:
Just moved the mean.go and the mean_test.go under the directory of maths

Reply:
I can still see the file present. You need to remove this file and commit the removal to git and push your changes. Otherwise it will remain in the source control.